docs/user_interface.rst

```diff
@@ -25,284 +25,416 @@ If you do processing or acquisition which produces new data, its associated disp
+
+.. _Metadata Panel:
+
+Metadata
+========
+The Metadata utility panel shows any metadata associated with the selected data item. The session info will be added as metadata to any live data acquired during a given session.
+
+.. _Output Panel:
 
-Tools
------
-.. tools: pointer, hand, line, rectangle, ellipse, point, line profile, interval, spot, wedge, band pass, array
-.. images: zoom options
-.. workspace: split h,v, 2x2, 3x2, 3x3, 4x3, 4x4, 5x4, select more panels, clear selected panels, reset workspace, close selected panels
+Output
+======
+The Output panel displays output text at the bottom of the window while running Nion Swift. This is useful for debugging the application.
 
-The Tools panel (:menuselection:`Window --> Tools`) allows you to select tools, adjust image zooming, and modify the workspace.
+.. _Sessions Panel:
 
-The tools available are the pointer, hand, line, rectangle, ellipse, point, line profile, interval, spot, wedge, band pass, and lattice tools. Some tools have keyboard shortcuts which can be seen by hovering over the tool.
+Session
+=======
+The Session panel allows you to edit the session info for the whole project rather than for a single data item. 
 
-The zoom buttons allow you to set raster image displays to fill the space with the image (Fill), fit the image to the space (Fit), set the pixel scaling to one data pixel per screen pixel (1:1), and set the pixel scaling to one data pixel per two screen pixels (2:1).
+.. image:: graphics/session_panel.png
+    :width: 321
+    :alt: Session Panel
 
-The workspace buttons allow you to split the workspace panels horizontally and vertically, or into grids of 2x2, 3x2, 3x3, 4x3, 4x4, 5x4. There is a button to expand the selected display panels. Pressing this button repeatedly allows you to select all of the display panels with a few clicks. There are also buttons to clear the contents of the selected display panels, close the selected display panels, and reset the workspace to a single display panel.
+The session info will be added as metadata to any live data acquired during a given session. A new session starts and global session info resets every time the Nion Swift is closed.
 
 .. _Task Panel:
 
 Task Panel
-----------
-.. table output from alignment
+==========
+The Task panel allows you to see the output from tasks such as microscope tuning. The output is often arranged into a table of data.
 
-The Task panel (:menuselection:`Window --> Task Panel`) allows you to see the output from tasks such as microscope tuning. The output is often arranged into a table of data.
+.. _Tool Panel:
+
+Toolbar
+=======
+The Toolbar provides quick access to several options for the workspace. You can select items, move images, add graphics, create masks, choose image scale options, and edit the layout and content of display panels.
```

Comment:
You might want to mention that some tools have key overloads and you can see the key overloads by hovering over the tool. And returning to the pointer tool is 'e'. A display panel has to be selected for the key overloads to work.